src/components/game-setup/GameStepper.js

```diff
@@ -0,0 +1,137 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Stepper from '@mui/material/Stepper';
+import Step from '@mui/material/Step';
+import StepButton from '@mui/material/StepButton';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import Customize from './stepper-components/Customize';
+import GameType from './stepper-components/GameType';
+import Lobby from './stepper-components/Lobby';
+
+const steps = ['Choose Game Type', 'Add Friends', 'Customize and Play'];
+
+export default function HorizontalNonLinearStepper() {
+  const [activeStep, setActiveStep] = React.useState(0);
+  const [completed, setCompleted] = React.useState({});
+
+  const totalSteps = () => {
+    return steps.length;
+  };
```

Comment:
Could just make this a variable rather than a function.

```suggestion
  const totalSteps = steps.length;
```